### PR TITLE
Increased weight of zombie hulk

### LINKED ISSUE #40076: Big zombies' corpses have unrealistically low density
# Describe the bug

Exactly what it says on the tin, their corpses are at 0.2-0.4 kg/L. What are they made of, feathers?

# Steps To Reproduce

Go find/summon skelly jugg or kevlar hulk, kill them, see their corpses are 200/875 and 200/625 kg/L respectively.

# Expected behavior

Regular zeds (and survivors, for that matter) are somewhat more dense than water. Seeing as hulks and skelejuggs are just big zombies with additional armor (which, too, shouldn't be weightless), their corpses should either take less space or weigh more. Which one exactly is, I think, for lorekeepers to decide.

# Versions and configuration

- OS: Windows 7x64
- Game Version: 0.E-1420-g7e4b8ce [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Modular Turrets [modular_turrets],
    Salvaged Robots [Salvaged_Robots],
    Aftershock [aftershock],
    Alternative Map Key [alt_map_key],
    sees-player icon, retrodays [sees_player_retro],
    Stats Through Skills [StatsThroughSkills],
    Magiclysm [magiclysm]
]

### PR DESCRIPTION
**Pull request by Night-Pryanik**
#### Summary
Bugfixes "Increased weight of zombie hulk"

#### Purpose of change
* Closes #40076.

#### Describe the solution
Increased weight of zombie hulk from 200 to 1000 kilograms.

#### Describe alternatives you've considered
None.

#### Testing
Debug-spawned zombie hulk, killed it, checked its corpse's weight.

#### Additional context
None.

### CODE REVIEW

**Comment by SirPendrak** [comment]
Mass of 200kg seems very low.

**Comment by Maleclypse** [comment]
I'm not seeing the message logs for the errors but I'm wondering if Hulk weight is an important part of the failing tests?

```
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> -------------------------------------------------------------------------------
(~[slow] ~[.],starting_items)=> Martial_art_technique_conditionals
(~[slow] ~[.],starting_items)=>   Test sweep
(~[slow] ~[.],starting_items)=> -------------------------------------------------------------------------------
(~[slow] ~[.],starting_items)=> ../tests/martial_art_test.cpp:163
(~[slow] ~[.],starting_items)=> ...............................................................................
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> ../tests/martial_art_test.cpp:180: FAILED:
(~[slow] ~[.],starting_items)=>   CHECK( !dude.evaluate_technique( tec, target_2, dude.used_weapon(), false, false, false ).has_value() )
(~[slow] ~[.],starting_items)=> with expansion:
Error: (~[slow] ~[.],starting_items)=>   false
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> -------------------------------------------------------------------------------
(~[slow] ~[.],starting_items)=> Martial_art_technique_conditionals
(~[slow] ~[.],starting_items)=>   Test knockback
(~[slow] ~[.],starting_items)=> -------------------------------------------------------------------------------
(~[slow] ~[.],starting_items)=> ../tests/martial_art_test.cpp:216
(~[slow] ~[.],starting_items)=> ...............................................................................
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> ../tests/martial_art_test.cpp:235: FAILED:
(~[slow] ~[.],starting_items)=>   CHECK( !dude.evaluate_technique( tec, target_2, dude.used_weapon(), false, false, false ).has_value() )
(~[slow] ~[.],starting_items)=> with expansion:
Error: (~[slow] ~[.],starting_items)=>   false
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> Log messages during failed test:
```

**Comment by Night-Pryanik** [comment]
I'm not sure why are we checking for failing a sweep and knockback techniques versus big zombies. I see two solutions here:
- require success instead of fail;
- find some other big zombie (with wrong json-defined size to pass the test?).

**Comment by Maleclypse** [comment]
> I'm not sure why are we checking for _failing_ a sweep and knockback techniques versus big zombies. I see two solutions here:
> 
>     * require success instead of fail;
> 
>     * find some other big zombie (with wrong json-defined size to pass the test?).

If I understand correctly we are checking to make sure that big monsters can't be swept or thrown.  Could the size be made equal or greater to 5?

**Comment by Night-Pryanik** [comment]
>  Could the size be made equal or greater to 5?

Tests failing not because of wrong size. It's because game says that it's possible to sweep or throw a big monster, but tests require it shouldn't be possible. Contradiction.

**Comment by RenechCDDA** [comment]
This changes the hulk's size from `HUGE` to `LARGE`. Maybe we could just say six**teen** men instead?

I don't see a lot of value in changing around the game balance and test coverage because a description wasn't 100% accurate.

**Comment by esotericist** [comment]
the hulk's size should not be changing here. increase the weight to fix the density, change the description if the inconsistency really bothers you, but it should be kept at the original size.

> Tests failing not because of wrong size. It's because game says that it's possible to sweep or throw a big monster, but tests require it shouldn't be possible. Contradiction.

there wasn't a contradiction until you adjusted the size *then* adjusted the tests that were there to make sure the size was expected.

this sort of thing is *exactly why the tests are there*, so that someone doesn't go changing parameters in a way that change game balance casually. just hammering the tests to be quiet without stopping to ask "should i be changing the game balance on this?" is not the way to go about things.

File: data/json/monsters/zed_misc.json [hunk 1]
```diff
@@ -390,7 +390,7 @@
     "copy-from": "mon_zombie_base",
     "diff": 5,
     "volume": "875 L",
-    "weight": "200 kg",
+    "weight": "1000 kg",
```
Comment (Maleclypse) [inline data/json/monsters/zed_misc.json:393]:
```suggestion
    "weight": "1000 kg"
```
New commit to fire the new required clang test

File: data/json/monsters/zed_misc.json [hunk 2]
```diff
@@ -390,7 +390,7 @@
     "copy-from": "mon_zombie_base",
     "diff": 5,
     "volume": "875 L",
-    "weight": "200 kg",
+    "weight": "1000 kg"
```
Comment (Maleclypse) [inline data/json/monsters/zed_misc.json:393]:
```suggestion
    "weight": "1000 kg",
```

**Comment by Maleclypse** [comment]
Testing closing to kick the delayed test

**Pull request merged**